### PR TITLE
docs(toolbar): spruce up the toolbar component examples

### PR DESCRIPTION
**Pull request by livvielin**
Screenshot of basic, colored, and multi-row toolbars included in toolbar-overview-example:
Screen Shot 2020-03-20 at 10 31 11 AM

### CODE REVIEW

**Comment by livvielin** [comment]
Looking to get initial feedback on this. Is this the sort of thing you're looking for to spruce examples up? Should there be more styling or complex interactions? I wasn't sure since the demo pages seem to start with "Basic usage" on the overview page and keep the fancier examples in the demo section.

File: src/components-examples/material/toolbar/toolbar-overview/toolbar-overview-example.html [hunk 1]
```diff
@@ -1 +1,13 @@
-<mat-toolbar>My App</mat-toolbar>
+<mat-toolbar>
+	<button mat-icon-button class="example-icon" aria-label="Example icon-button with menu icon">
```
Comment (jelbourn) [inline src/components-examples/material/toolbar/toolbar-overview/toolbar-overview-example.html]: Indent should be +2 rather than +4

Comment (livvielin) [reply src/components-examples/material/toolbar/toolbar-overview/toolbar-overview-example.html]: Converted to +2

**Comment by livvielin** [comment]
Awesome, thanks for the feedback! I've added a couple more examples to toolbar-overview-example and then added it to the toolbar demo page in dev-app.

**Comment by livvielin** [comment]
@jelbourn Is there anything else I should add to this PR or change?

File: src/components-examples/material/toolbar/toolbar-overview/toolbar-overview-example.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import {Component} from '@angular/core';
 
 /**
- * @title Basic toolbar
+ * @title Overview toolbar
```
Comment (jelbourn) [inline src/components-examples/material/toolbar/toolbar-overview/toolbar-overview-example.ts:4]: How about "Toolbar overview"?

Comment (livvielin) [reply src/components-examples/material/toolbar/toolbar-overview/toolbar-overview-example.ts]: Sounds good, updated. Thanks!

**Pull request merged**